inject/android.js

```diff
@@ -8,11 +8,11 @@ function init () {
     if (isGloballyDisabled(processedConfig)) {
         return
     }
-    // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
```

Comment:
I fixed a TypeScript issue for the files I'm changing, but that caused TypeScript to complain that these unrelated `@ts-expect-error`s were no longer needed... so that's why I had to remove them all!